twinPicks/jsonld-contexts/twinPicks-compound.jsonld

```diff
@@ -1,11 +1,17 @@
 {
   "@context": [
-    "https://easy-global-market.github.io/ngsild-api-data-models/dcat/jsonld-contexts/dcat-compound.jsonld",
+    "https://easy-global-market.github.io/c2jn-data-models/jsonld-contexts/datacenter-equipments.jsonld",
+    "https://easy-global-market.github.io/ngsild-api-data-models/dcat/jsonld-contexts/dcat.jsonld",
+    "https://easy-global-market.github.io/c2jn-data-models/jsonld-contexts/c2jn-smart-city-compound.jsonld",
+    "https://easy-global-market.github.io/ngsild-api-data-models/archive/digital-twins/jsonld-contexts/dcat.jsonld",
```

Comment:
two `dcat.jsonld`?

Reply:
Don't sure which one to keep. But we should stop using the /archive anyways.